src/pages/index.js

```diff
@@ -10,9 +10,16 @@ function Home() {
 
   return (
     <Layout title="Homepage" description="Avalanche Dev Docs">
-      <main>
+      <main className={styles.main}>
         <br />
-        <h1 align="center" style={{ fontWeight: "750" }}>
+        <h1
+          align="center"
+          style={{
+            fontWeight: "700",
+            marginBottom: "0px",
+            fontSize: "x-large",
```

Comment:
what's `x-large`?

Reply:
honestly not sure, was a suggestion from my css plugin

Reply:
Docusaurus default styling if inherited from the [Infima](https://infima.dev/) styling framework so values might be from there?